code/modules/experisci/experiment/types/scanning_machinery.dm

```diff
@@ -18,9 +18,14 @@
 		return FALSE
 
 	var/obj/machinery/machine = target
+	//this checks for stock parts as objects inside the machine, will be obsolete when stock parts become datumised; please remove this then
```

Comment:
FWIW Not all stock parts can be datumized out, power cells specifically

Reply:
i'll adjust the comment so it doesn't get misinterpreted in the datumised future